build-tools/heroku/generate-typescript

```diff
@@ -1,73 +1,6 @@
 #!/bin/bash
 set -eu
-# Recompile the TypeScript and generate a minified JavaScript bundle using esbuild.
-#
-# We commit the generated JavaScript bundle to minimize the burden on contributors--
-# they don't need to install any of the NPM/esbuild tools and remember to run build
-# scripts if they don't need to work on the browser code.
-#
-# We minimize the generated bundle so that contributors aren't tempted to accidentally
-# edit straight in the JavaScript.
-#
-#----------------------------------------------------------------------
-#
-# This script does two similar, but different jobs:
-#
-# In normal build mode, we first compile TypeScript -> JavaScript using the
-# TypeScript compiler (TSC), and then bundle it all into one file using esbuild.
-# The reason we do this is because TSC can translate certain desirable idioms
-# that are are not supposed in old browsers, like `async`, `?.` and `??` to
-# plain old JavaScript. `esbuild` does not do that, which is why have TSC do
-# this job first.
-#
-# We can also run in `--watch` mode, in which case we bypass the TypeScript
-# compiler and only run esbuild. This is because the TypeScript compiler is
-# rather slow (think 5-10 seconds to do a compilation), and we want to
-# optimize for iteration speed in that case. That does mean that the generated
-# JavaScript is slightly different between those cases--however, the final
-# JavaScript will always be generated on deploy using the official generation
-# method.
```

Comment:
Ahhh is this why you kept the bash files even though we could run them with `doit`?

Reply:
Oh, no. This functionality will actually be gone. I don't know how many people use it...

Reply:
I use it :c